rust/processor/src/schema.rs

```diff
@@ -268,6 +270,25 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    current_fungible_asset_balances (storage_id) {
```

Comment:
One new unique attribute in fungible asset is whether the entire store can be transferred or not. May be we can add a quick attribute for this (is_store_transferrable)?

Reply:
Wait really? This is a feature now? Interesting, can you point me to the PR lol.

Reply:
It's always been the case. The fungible store is transferrable if it's non-primary (only primary store objects are frozen)

Reply:
Oh then that's fine right? !is_primary = is_store_transferrable

Reply:
is_store_transferrable for non-primary depends on the actual object itself being transferrable and might always be true

Reply:
So this is the ungated_transfer thign? Tbh I don't think that it's useful from a indexer standpoint. Move can get this field in some ways, and downstream shouldn't care about it.

Reply:
It might be applicable here for activities though because I transfer ownership of a store of 1000 SomeAsset to you, it's the same as giving you 1000 SomeAsset?

Reply:
Oh. I thought that we didn't allow this unless for "burning". We specifically talked about how the right way to transfer asset is a withdraw + deposit. Transferring the entire wallet is not ideal. Is this...not true anymore? It might mess things up for downstream, e.g. wallet cc @kent-white 

I Btw wallet did decide to not care about non primary wallet.

Reply:
Secondary stores can be created by anyone and still transferrable. I think for "burning", you're referring to a slightly different case of soul bound token v2 and soul bound fungible store (e.g. non-transferrable like primary stores). We decided last time to only do this for token v2 for now as it's messy to allow generate "burning" of non-transferrable objects, especially when it comes to fungible assets.

Reply:
Wait what? No we said that we would also "burn" fungible assets.

Reply:
Just transfer the primary store to 0xgarbage. I'm not crazy right that this is what we said last time right @lightmark @davidiw @kent-white @CapCap

Reply:
Nope we never talked about fungible assets. Transferring the primary store to 0xsomething is bad as it means youll never be able to receive that tokens (eg you realize later is not scammy). It also breaks the accounting in any defi modules that rely on locking down transfers so they can update their accounting if transfers do happen. Transferring primary store breaks everything there